docs/src/usage/filter.md

```diff
@@ -0,0 +1,58 @@
+# Filtering
+
+Filtering in `Neuroimaging.jl` is performed using the `DSP.jl` backend.
+
+There are two main ways to use the filter function:
+1) Modality specific filter
+2) Custom defined filter
+
+Let's load some data and apply the filter
+
+```@example filter
+using DisplayAs # hide
+using Neuroimaging, DataDeps, StatsBase
+
+data_path = joinpath(
+    datadep"ExampleSSR",
+    "Neuroimaging.jl-example-data-master",
+    "neuroimaingSSR.bdf",
+)
+
+s = read_EEG(data_path)
+s.data = s.data .- StatsBase.mean(s.data,dims=1) # remove DC offset for better plotting
+```
+
+
+## 1) Modality specific filter
+We offer high and lowpass filter with reasonable defaults for each modality (e.g. `GeneralEEG` a FIR filter,`SSR` a filtfilt Butterworth)
+
+```@example filter
+s_hp = filter_highpass(s,2)
+s_lp = filter_lowpass(s,5) # extreme value to show LP effect in plot
+
+plot(s.data[:,1],label="raw")
```

Comment:
Don't index data directly (see comment below). To plot just a single channel use the `keep_channel!` function